src/gurobi_optimods/opf/grbformulator_ac.py

```diff
@@ -769,6 +816,52 @@ def lpformulator_ac_create_constraints(alldata, model):
 
     logger.info(f"    {count} balance constraints added.")
 
+    if alldata["skipjabr"] and alldata["use_ef"]:
+        # voltage magnitude bounds
+        logger.info("  Adding voltage magnitude constraints.")
+        count = 0
+        for j, bus in buses.items():
+            expr = gp.QuadExpr()
+            expr.add(evar[bus] * evar[bus] + fvar[bus] * fvar[bus])
```

Comment:
Why not just `expr = evar[bus] * evar[bus] + fvar[bus] * fvar[bus]`?

Reply:
I just adapted the old code, I can change it.